README.md

```diff
@@ -84,7 +84,7 @@ Want to contribute? Here is the source code https://github.com/linkedin-faq
 | [Google Analytics](google-analytics/google-analytics-quiz.md)                                     |                  |                                                                                                                                                                                                                                                                                  | 82        | 82      |                                                                                                                                                                       |
 | [Google Cloud Platform (GCP)](google-cloud-platform/google-cloud-platform-quiz.md)                |                  |                                                                                                                                                                                                                                                                                  | 52        | 50      | [@antra0497](https://github.com/antra0497)                                                                                                                            |
 | [Hadoop](hadoop/hadoop-quiz.md)                                                                   |                  |                                                                                                                                                                                                                                                                                  | 66        | 54      |                                                                                                                                                                       |
-| [HTML](html/html-quiz.md)                                                                         |                  |                                                                                                                                                                                                                                                                                  | 110       | 106     | [@declarck](https://github.com/declarck)                                                                                                                              |
+| [HTML](html/html-quiz.md)                                                                         |                  |                                                                                                                                                                                                                                                                                  | 129       | 106     | [@declarck](https://github.com/declarck)                                                                                                                              |
```

Comment:
The number of questions answered is 128, not 106

Reply:
Done